src/libs/actions/PaymentMethods.ts

```diff
@@ -83,7 +83,7 @@ function getMakeDefaultPaymentOnyxData(
                   onyxMethod: Onyx.METHOD.MERGE,
                   key: ONYXKEYS.USER_WALLET,
                   value: {
-                      walletLinkedAccountID: bankAccountID || fundID,
+                      walletLinkedAccountID: bankAccountID ?? fundID,
```

Comment:
This is not equivalent for `bankAccountID` being `0`

Reply:
Yeah, you're likely right... Unfortunately, we often use stuff like `''` to mark an "undefined" string, so I wouldn't be surprised if `0` was used as a special "undefined" value somewhere. I'm aware of the mentioned lint.

We can stick with `??`.